lib/tests/test_git.rs

```diff
@@ -159,14 +159,14 @@ fn test_import_refs() {
         view.get_remote_branch("main", "git"),
         &RemoteRef {
             target: RefTarget::normal(jj_id(&commit2)),
-            state: RemoteRefState::Tracking,
+            state: RemoteRefState::Tracking
```

Comment:
nit: unrelated formatting change in `test_git.rs`.

Reply:
Fixed. Not sure how that happened.